src/openms/include/OpenMS/METADATA/PeptideIdentification.h

```diff
@@ -145,6 +145,15 @@ namespace OpenMS
     /// sets the experiment label for this identification
     void setExperimentLabel(const String& type);
 
+    /// returns the spectrum reference for this identification. Currently it should
+    /// almost always be the full native vendor ID.
+    // TODO make a mandatory data member, add to idXML schema, think about storing the
+    //  extracted spectrum "number" only!
```

Comment:
We might need to Consider that there might be multiple spectrum=1 if different detectors are used etc. we recently had a similar issue

Reply:
Ok then I think one should raise an issue at sage/percolator regarding the format.